src/components/Person/Person.jsx

```diff
@@ -1 +1,14 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  const partner = sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {age && <p className="Person__age">{`I am ${age}`}</p>}
+      <p className="Person__partner">
+        {isMarried ? `${partnerName} is my ${partner}` : `I am not married`}
```

Comment:
<img width="1088" alt="image" src="https://github.com/mate-academy/react_person/assets/86789161/e978c18d-5406-421d-b4fa-a7e1fce17725">

Or even better create a variable for this